apim-lab/6-analyticsMonitoring/apimanagement-6-2-ApplicationInsights.md

```diff
@@ -75,7 +75,7 @@ A failed request is a request which:
 
 ### Generating Test Requests
 
-Any request you make to the Color API in APIM will be subject to being received by Application Insights (recall the 100% sampling). To generate a large amount of requests quickly, you can invoke the API via the [Color](https://markcolorweb.azurewebsites.net) website. As we are presently caching the output for 15 seconds, you may get a lot of requests with the same color. Please feel free to disable the output caching in the Color API if you would like to see more variety.
+Any request you make to the Color API in APIM will be subject to being received by Application Insights (recall the 100% sampling). To generate a large amount of requests quickly, you can invoke the API via the [Color](https://colors-web.azurewebsites.net/) website. As we are presently caching the output for 15 seconds, you may get a lot of requests with the same color. Please feel free to disable the output caching in the Color API if you would like to see more variety.

```

Comment:
We can remove all trailing slashes for the colors website throughout. There's no need to have it in the URL.